android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -308,21 +310,17 @@ public void finishNativeInitialization() {
         }
         BraveSyncReflectionUtils.showInformers();
 
+        if (!BravePrefServiceBridge.getInstance().hasPathP3AEnabled()) {
+            BravePrefServiceBridge.getInstance().setP3AEnabled(false);
+        }
+
         if (BraveConfig.P3A_ENABLED) {
-            // // P3A informer should be shown not for the updated users only
-            // if (!PackageUtils.isFirstInstall(this)) {
-            //     BraveP3AInformers.show();
-            // } else {
-            //     // On the first run P3A is must be set from the onboarding wizard
-            // }
-            // Uncomment ^ when onboarding UI step will be ready for P3A
-
-            // Until we don't have P3A onboarding, P3A will be disabled,
-            // but available for activating through settings
             if (!BravePrefServiceBridge.getInstance().hasPathP3AEnabled()) {
                 BravePrefServiceBridge.getInstance().setP3AEnabled(false);
             }
```

Comment:
do we need that duplicated code here? we have it above as well before the condition check https://github.com/brave/brave-core/pull/7550/files#diff-448df90d6aa5a20513ca5dba42a4e428dfd22f05577b904f8fe7986c40ab90faR313